src/main/java/sorald/processor/ToStringReturningNullProcessor.java

```diff
@@ -0,0 +1,30 @@
+package sorald.processor;
+
+import sorald.Constants;
+import sorald.annotations.IncompleteProcessor;
+import sorald.annotations.ProcessorAnnotation;
+import spoon.reflect.code.CtReturn;
+import spoon.reflect.declaration.CtMethod;
+import spoon.reflect.visitor.filter.TypeFilter;
+
+@IncompleteProcessor(description = "does not fix null returning clone()")
+@ProcessorAnnotation(
+        key = 2225,
+        description = "\"toString()\" and \"clone()\" methods should not return null")
+public class ToStringReturningNullProcessor extends SoraldAbstractProcessor<CtReturn<?>> {
+
+    @Override
+    protected boolean canRepairInternal(CtReturn<?> candidate) {
+        CtMethod parentMethod = candidate.getParent(new TypeFilter<>(CtMethod.class));
```

Comment:
Two comments on this line: avoid raw generics, and `CtElement.getParent` has a convenience method for getting an element of a specific type.

```suggestion
        CtMethod<?> parentMethod = candidate.getParent(CtMethod.class);
```